### ISSUE TITLE
Primitives without a fixed size don't work

### ISSUE THREAD

**Issue by adetaylor** [google/autocxx#156]
Fails:

```rust
#[test]
fn test_ulong() {
    let hdr = indoc! {"
    unsigned long daft(unsigned long a) { return a; }
    "};
    let rs = quote! {
        assert_eq!(ffi::daft(34.into(), 34));
    };
    run_test("", hdr, rs, &["daft"], &[]);
}
```

This attempts to pass this into cxx:
```rust
pub fn daft(a: ::std::os::raw::c_ulong) -> ::std::os::raw::c_ulong;
```

**Comment by adetaylor** [comment]
Related: https://github.com/dtolnay/cxx/issues/254

**Comment by adetaylor** [comment]
Fixed by #157.
